Jetpack Compose Material3 1.3.0-beta05

igordmn opened the pull request:
Copied from b15c04590b904b5e626f97976a19074a5d26a66b

igordmn committed:
Update Jetpack Compose Material3 to 1.3.0-beta05

Copied from b15c04590b904b5e626f97976a19074a5d26a66b

igordmn committed:
artifactRedirecting.androidx.compose.material3.version=1.3.0-beta05

MatkovIvan commented on compose/material3/material3/src/commonMain/kotlin/androidx/compose/material3/Slider.kt:
Our `androidx.compose.material3.SliderTest` fails. Probably we need to copy the changes from Android one

igordmn replied:
Fixed https://github.com/JetBrains/compose-multiplatform-core/pull/1479/commits/767a88e870433ec2bd2871620009e8260b97db93

igordmn committed:
Restore `material3\build.gradle` to jb-main state

igordmn committed:
Fix `SliderTest.changingOnValueChangeFinishedDoesNotTriggerFinish`

Slider has a bigger semantics/test size in 1.3.0-beta05 - it goes beyond window bounds.

Pointer events aren't allowed beyond bounds by the common `ui-test` logic, so we just change testing pointer events.

(tests in the android's SliderTest are also always started from the center)

MatkovIvan commented on compose/material3/material3/src/commonMain/kotlin/androidx/compose/material3/internal/AccessibilityUtil.kt:
```diff
@@ -0,0 +1,48 @@
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.offset
+
+@VisibleForTesting internal val HorizontalSemanticsBoundsPadding: Dp = 10.dp
```
Does it work well on all platforms?

igordmn replied:
I checked Desktop usual interaction (works the same) and VoiceOver borders (they are increased and looks better)
